.coderabbit.yml

```
@@ -0,0 +1,16 @@
+# Review settings
+reviews:
+  request_changes_workflow: true
+  high_level_summary: true
+  poem: false
+  review_status: false
+  collapse_walkthrough: true
+
+# Enable auto review for non-draft PRs
+auto_review:
+  enabled: true
+  drafts: false
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)